docs/querying/basics.md

```diff
@@ -153,6 +155,9 @@ A workaround for this restriction is to use the `__name__` label:
 All regular expressions in Prometheus use [RE2
 syntax](https://github.com/google/re2/wiki/Syntax).
 
+It is not possible to specify a specific time or age in PromQL when querying instant vectors,
```

Comment:
This is not quite true, as the `@` operator (described below) allows setting an absolute evaluation timestamp for an instant vector selector, a range vector selector, or a subquery expression. I would maybe just mention in the initial paragraph (the one with your other edits) that by default, the sample selection happens relative to the query evaluation timestamp?

Reply:
Ooh, that'd be _great_ to surface more clearly in the docs too.

I'll amend the wording.